ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Superuser/tasks/main.yml

```diff
@@ -4,6 +4,7 @@
 #############
 - name: Create Superuser account - zeus
   action: user name=zeus state=present shell=/bin/bash
+  ignore_errors: yes
```

Comment:
Why are we ignoring errors here?

Reply:
This was copied from the Jenkins user step. The create user role fails on Solaris and macOS because of home directory issues but does actually create the user so the skip step works around this.

Reply:
What's the issue on macos? We could set a variable for the home dir location and pass it into this operation with `home=/export/home/zeus` or wherever else it is (defaulting to `/home/zeus`. We run the risk of playbooks completing without anyone realising that it hasn't done the operations if we ignore errors here.

Reply:
updated